easybuild/easyconfigs/d/DRAGMAP/DRAGMAP-1.3.0-GCC-13.2.0.eb

```diff
@@ -0,0 +1,39 @@
+# #include <cstdint> added to deal with 'uint64_t' problem, sed command removed
+# Author: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'ConfigureMake'
+
+name = 'DRAGMAP'
+version = '1.3.0'
+
+homepage = 'https://github.com/Illumina/DRAGMAP'
+description = 'Dragmap is the Dragen mapper/aligner Open Source Software.'
+
+toolchain = {'name': 'GCC', 'version': '13.2.0'}
+
+source_urls = ['https://github.com/Illumina/DRAGMAP/archive/refs/tags/']
+sources = ['%(version)s.tar.gz']
+patches = ['%(namelower)s-%(version)s-include.patch']
+checksums = [
+    {'1.3.0.tar.gz': 'baf546c2f53b13e2bce4c937201e38459f3ce2f1069722ca670d8726a4bae224'},
+    {'dragmap-1.3.0-include.patch': '60c364696dd98c70023696a42419a4251bcc737535e44217c9ce03bdc8084338'},
+]
+
+dependencies = {
+    ('Boost', '1.83.0'),
+    ('googletest', '1.14.0'),
+}
+
+skipsteps = ['configure']
```

Comment:
@sassy-crick  It would probably make sense to keep the
```
-prebuildopts = 'sed -i "s|-msse4.2 -mavx2|$CXXFLAGS|g" config.mk &&'
```
line from previous version of this since it's the same version

Reply:
I think you are right. I added that line again, after my local re-build worked.